gradle.properties

```diff
@@ -1,5 +1,5 @@
 version=1.75-SNAPSHOT
-groovyVersion=2.4.11
+groovyVersion=2.4.17
```

Comment:
Groovy 2.4.11 is used in Jenkins versions 2.61 - 2.142 - see https://jenkins.io/changelog/.

As 2.121 is the minimum Jenkins version for this plugin, the Groovy version should not be changed yet.